easybuild/easyconfigs/p/polars/polars-0.20.2-gfbf-2023a.eb

```diff
@@ -193,7 +193,8 @@ checksums = [
     {'jemallocator-0.5.4.tar.gz': 'a0de374a9f8e63150e6f5e8a60cc14c668226d7a347d8aee1a45766e3c4dd3bc'},
     {'jobserver-0.1.27.tar.gz': '8c37f63953c4c63420ed5fd3d6d398c719489b9f872b9fa683262f8edd363c7d'},
     {'js-sys-0.3.66.tar.gz': 'cee9c64da59eae3b50095c18d3e74f8b73c0b86d2792824ff01bbce68ba229ca'},
-    {'jsonpath_lib-0.3.0.tar.gz': 'dfb5eb7f47b5b5a7b35262f7b41787a785958001a23ff039cdae926396c6af96'},
+    {'jsonpath_lib-0.3.0-24eaf0b4416edff38a4d1b6b17bc4b9f3f047b4b.tar.gz':
+     'dfb5eb7f47b5b5a7b35262f7b41787a785958001a23ff039cdae926396c6af96'},
```

Comment:
part of the issue is that this repo is cloned with its `.git`, which makes the checksum worthless

Reply:
https://github.com/easybuilders/easybuild-framework/issues/4793

Reply:
This is a source from github, isn't it? So for the crates can't we "rewrite" them to archive downloads instead of git checkouts as we do now? I mean it is us anyway creating a "source" entry from a crate tuple

Reply:
partial improvement to the checksum verification failures: https://github.com/easybuilders/easybuild-framework/pull/4798

Reply:
final fix related to the checksums:
1. this checksum definition needs to be changed to:
  ```
  {'jsonpath_lib-0.3.0-24eaf0b4.tar.xz': 'cc7ed586a3047975d1d9c2ab8e2c555403fd73ee94358746476053610dc7024f'},
  ```
2. cargo easyblock needs a little update to not use `tar.xz` on these sources. I guess I'll PR this change to @Flamefire PR https://github.com/easybuilders/easybuild-easyblocks/pull/3654